build.sbt

```
@@ -36,6 +36,8 @@ lazy val commonSettings = Seq(
   }
 )
 
+lazy val warningSuppression = Seq(scalacOptions += "-Wconf:msg=APIs in chisel3.internal:s,msg=Importing from firrtl:s,msg=migration to the MLIR:s")
```

```suggestion
lazy val warningSuppression = Seq(
  scalacOptions += "-Wconf:" + Seq(
    "msg=APIs in chisel3.internal:s",
    "msg=Importing from firrtl:s",
    "msg=migration to the MLIR:s"
  ).mkString(",")
)
```
I think we should write this in a way that highlights each rule being applied.